app/controllers/records_controller.rb

```diff
@@ -1,8 +1,22 @@
 class RecordsController < ApplicationController
   include RecordsControllerBehavior
 
+  def ingest_options
+    @templates = [["Raw (no template)", nil]] + template_class.all.collect {|t| [t.title, t.id]}
```

Comment:
Can you extract `template_class.all.collect {|t| [t.title, t.id]}` to an instance method? Then in that instance method, more efficient would be `template_class.all.pluck(:title, :id)`